### PR TITLE
PHPDoc fix

### PR DESCRIPTION
**Pull request by OskarStark**
ping @core23

### CODE REVIEW

File: Model/Media.php [hunk 1]
```diff
@@ -63,7 +63,7 @@
     protected $height;
 
     /**
-     * @var decimal
+     * @var floar
```
Comment (core23) [inline Model/Media.php]: Float please ;)

Comment (OskarStark) [inline Model/Media.php]: damn :)

**Comment by OskarStark** [comment]
done @core23

**Comment by core23** [comment]
Thank you @OskarStark

**Comment by core23** [comment]
Did you create a branch on the repository? oO

**Comment by OskarStark** [comment]
Yes, edited via the editor on github :smile:

**Pull request merged**